app/models/miq_user_role.rb

```diff
@@ -75,26 +75,29 @@ def self.with_allowed_roles_for(user_or_group)
   end
 
   def self.seed
-    seed_from_array(YAML.load_file(FIXTURE_YAML))
+    roles = all.index_by(&:name)
+    seed_from_array(roles, YAML.load_file(FIXTURE_YAML))
```

Comment:
I know it's not related to your change, but it's bothering me that we have duplicate code here.  Do you think it would make sense to move `FIXTURE_YAML` into the `FIXTURE_PATH`?